framework/tlvf/AutoGenerated/include/tlvf/ieee_1905_1/tlvDeviceInformation.h

```diff
@@ -63,9 +53,37 @@ class tlvDeviceInformation : public BaseClass
         eTlvType* m_type = nullptr;
         uint16_t* m_length = nullptr;
         sMacAddr* m_mac = nullptr;
-        uint8_t* m_info_length = nullptr;
-        sInfo* m_info = nullptr;
-        size_t m_info_idx__ = 0;
+        uint8_t* m_local_interface_list_length = nullptr;
```

Comment:
Why have a separate variable for this? Isn't the local interface length simply `m_local_interface_list_vector.size()`?

Reply:
This code is auto-generated. 
Regarding this, I had some question to ask which maybe @tomereli could answer: the definition of the array of nested structures in this TLV is pretty much the same as in SupportedService TLV but auto-generated code to deal with the list of elements is different. Is there any flag to tune how to generate the code to manage the array defined in .yaml?

Reply:
Ah, I didn't notice I was looking at auto-generated code, sorry :-)